lib/Dialect/TTIR/IR/TTIROps.cpp

```diff
@@ -45,6 +45,24 @@ ::mlir::LogicalResult mlir::tt::ttir::ClampOp::verify() {
   return success();
 }
 
+//===----------------------------------------------------------------------===//
+// RoundOp/ RoundNearestEvenOp
+//===----------------------------------------------------------------------===//
+
+::mlir::LogicalResult mlir::tt::ttir::RoundOp::verify() {
+  if (getDecimals() == 0) {
```

Comment:
Even if this was true (which I doubt), this is a really bad design. If a type can have just one value then it's isomorphic to void and shouldn't be a parameter at all, it should be explicitly set during lowering if appropriate op on lower level dialect requires it.